Resources/Prototypes/_RMC14/Roles/Loadouts/Items/RoleSpecific/synthetic.yml

```
@@ -0,0 +1,299 @@
+# Synthetic
+
+# UNMC Jumpsuits
+- type: loadout
+  id: ScrubsBlueRMC
+  equipment:
+    jumpsuit: CMScrubsBlue
```

```suggestion
- type: loadout
  id: SynthUniformRMC
  equipment:
    jumpsuit: RMCJumpsuitSynthetic

- type: loadout
  id: ScrubsBlueRMC
  equipment:
    jumpsuit: CMScrubsBlue
```
Have the synthetic jumpsuit in loadouts instead of starting gear. Loadouts can then completely handle jumpsuits for the character